### PR TITLE
add cloudsecurity_cdr to the integrations categories

### PR DESCRIPTION
**Pull request by CohenIdo**
## What does this PR do?
Introduces a new integration category - `cloudsecurity_cdr`
- https://github.com/elastic/security-team/issues/8667

## Checklist

- [ ] I have added test packages to [`test/packages`](https://github.com/elastic/package-spec/tree/main/test/packages) that prove my change is effective.
- [x] I have added an entry in [`spec/changelog.yml`](https://github.com/elastic/package-spec/blob/main/spec/changelog.yml).

## Related issues

-

### CODE REVIEW

**Comment by CohenIdo** [comment]
test integrations

File: spec/integration/manifest.spec.yml [hunk 1]
```diff
@@ -90,6 +90,7 @@ spec:
           - web_application_firewall
           - websphere
           - workplace_search
+          - cloudsecurity_cdr
```
Comment (jsoriano) [inline spec/integration/manifest.spec.yml:93]: Nit. Please add it in alphabetical order.

Comment (jsoriano) [inline spec/integration/manifest.spec.yml:93]: @nimarezainia are you ok with adding this category?

**Review by jsoriano — COMMENTED** [review]
Change looks good to me. Pinging Nima to confirmation about adding a new category.

Subcategory being added in https://github.com/elastic/package-registry/pull/1142/files.

File: spec/integration/manifest.spec.yml (hunk 1 above)
Comment (CohenIdo) [reply spec/integration/manifest.spec.yml]: done.

**Comment by CohenIdo** [comment]
cc: @snehsach19

**Comment by jsoriano** [comment]
Btw, categories are also listed in https://github.com/elastic/kibana/blob/b5225232d038ea8d29f4f21c4bd528209e1b365e/x-pack/plugins/fleet/common/types/models/package_spec.ts#L47

We may need a change in Kibana too. cc @kpollich

**Review by nimarezainia — APPROVED** [review]

**Comment by CohenIdo** [comment]
> Btw, categories are also listed in https://github.com/elastic/kibana/blob/b5225232d038ea8d29f4f21c4bd528209e1b365e/x-pack/plugins/fleet/common/types/models/package_spec.ts#L47 Update, also here: https://github.com/elastic/kibana/blob/b5225232d038ea8d29f4f21c4bd528209e1b365e/src/plugins/custom_integrations/common/index.ts#L15
> 
> We may need a change in Kibana too. cc @kpollich

Yes, we are aware for that, it's part of the issue we are tracking:
- https://github.com/elastic/security-team/issues/8667

**Review by jsoriano — APPROVED** [review]

File: spec/integration/manifest.spec.yml (hunk 1 above)
Comment (jsoriano) [reply spec/integration/manifest.spec.yml]: Not sure how these buttons are configured. This is probably coded in Kibana/Fleet.

**Pull request merged**